draft-sullivan-tls-implicit-ech.md

```diff
@@ -209,6 +280,19 @@ Supporting implicit ECH configurations limits the number of different ECH
 keys supported by a server on the same IP address since the outer SNI and
 config_id can no longer be used to choose the appropriate ECH configuration.
 
+Deployments can support key rotation using the following strategy:
+ * Add a new hashed public key to their ImplicitECHConfig
+ * Wait for the new ECHConfig to propagate (DNS TTL)
+ * Begin signing their retry_configs with the new public key.
+ * Remove the old public key hash from their ImplicitECHConfig.
+
+Deployments can handle key compromise by removing the affected public key
+from their ImplicitECHConfig.
+
+notAfter times SHOULD be at least 24 hours in the future and no more than 48 hours in the future. This allows
```

Comment:
I'd extend this a fair bit longer than 48 hours.  The value of the thing being protected here is far less than what a certificate protects and those last for longer.